firebase-inappmessaging/firebase-inappmessaging.gradle

```diff
@@ -106,7 +106,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-components:17.1.0'
     implementation project(':protolite-well-known-types')
     implementation 'com.google.android.datatransport:transport-api:3.0.0'
-    implementation 'com.google.firebase:firebase-datatransport:18.1.7'
+    implementation project(':firebase-datatransport')
```

Comment:
@rlazo Please confirm this is okay. To use the new Transport annotation, i need to add the direct dependency on the data transport.

Reply:
Yes, WAI.